src/foam/u2/crunch/CapabilityRequirementView.js

```diff
@@ -94,6 +95,9 @@ foam.CLASS({
             capability.id == this.capabilityId
         ) :
         undefined;
+      if ( mainCapability.length !== 0 ) {
+        mainCapability[0].copyFrom(this.theme.capabilityConfig[this.capabilityId]);
```

Comment:
Not sure how I feel about this, would prefer to do this automatically on theme load rather than having to add a copyFrom everywhere

Reply:
Would be a bigger problem in the case of capabilities that are not top level as well, potentially a client side dao decorator on capability DAO that intercepts selects and copies from theme